app/pods/publications/publication/case/template.hbs

```diff
@@ -3,8 +3,8 @@
   {{!--  DOSSIER PANEL  --}}
     <div class="auk-u-mb-3">
       {{#if this.isInscriptionInEditMode}}
-        <WebComponents::AuPanel data-test-publication-case-panel-case-editing class="auk-panel--editing">
-          <WebComponents::AuPanel::Header>
+        <Auk::Panel data-test-publication-case-panel-case-editing class="auk-panel--editing">
```

Comment:
deze auk-panel--editing wordt 2x gebruikt,  1 keer op de auk::Panel, 1 keer op de auk::Panel::Body (documents-filter).

waar hoort deze thuis en gaan we dit een optie maken in de auk component ?

Reply:
Zie https://github.com/kanselarij-vlaanderen/au-kaleidos-css/issues/9

Reply:
Die class zou alleszins al op `auk::Panel` moeten staan en niet op `auk::Panel::Body`. De mogelijkheid om dit als optie te kunnen doorgeven lijkt me wel handig 👍

Reply:
@MikiDi Ik kijk daar direct even voor.

Reply:
@MikiDi Zie comment binnen kanselarij-vlaanderen/au-kaleidos-css#9

Maak ik daarvoor dan nog een issue aan? (ik neem die dan wel op)